mlflow/server/js/src/components/artifact-view-components/ShowArtifactMapView.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { getSrc } from './ShowArtifactPage';
+import './ShowArtifactMapView.css';
+import { getRequestHeaders } from '../../setupAjaxHeaders';
+import 'leaflet/dist/leaflet.css';
+import L from 'leaflet';
+import icon from 'leaflet/dist/images/marker-icon.png';
+import iconRetina from 'leaflet/dist/images/marker-icon-2x.png';
+import iconShadow from 'leaflet/dist/images/marker-shadow.png';
+
+
+class ShowArtifactMapView extends Component {
```

Comment:
Could you please provide some basic rendering test for this new component?

Reply:
I don't work with javascript or React that much, so I don't think I can do that.
Maybe if you provide a test for `ShowArtifactTextView.js` I could base mine on that.

Reply:
@spadarian I think it'd be reasonable to base tests off of those in https://github.com/mlflow/mlflow/pull/1556/files#diff-b0bb5567cc37de50eeeffde66f9653b3

Reply:
@Zangr I took me a while but I managed to add a test. Thanks for your advice @smurching